lib/WorkflowMain.groovy

```diff
@@ -79,17 +79,4 @@ class WorkflowMain {
             System.exit(1)
         }
     }
-
```

Comment:
Why is this removed?

Reply:
Because this test is giving warnings the whole time and checks according to the NGS genome.